geopyspark/geotrellis/rdd.py

```diff
@@ -120,6 +120,32 @@ def to_numpy_rdd(self):
         ser = self.geopysc.create_tuple_serializer(result._2(), value_type=TILE)
         return self.geopysc.create_python_rdd(result._1(), ser)
 
+    def to_tiled_rdd(self, extent=None, layout=None, crs=None, tile_size=256,
```

Comment:
do you think `to_tiled_rdd` is more consistent or clearer than `to_tiled_layer` ?